test/e2e-v2/cases/postgresql/postgres-exporter/docker-compose.yml

```diff
@@ -20,23 +20,42 @@ services:
     extends:
       file: ../../../script/docker-compose/base-compose.yml
       service: oap
+    entrypoint: ['sh', '-c', '/download-mysql.sh && chmod 777 /skywalking/docker-entrypoint.sh && /skywalking/docker-entrypoint.sh']
```

Comment:
This is only needed when we use mysql driver, but psql does not use mysql driver, right?

```suggestion
```

Reply:
Yes, I forget to remove it.